cpp/src/ray/runtime/task/task_executor.cc

```diff
@@ -159,6 +161,11 @@ Status TaskExecutor::ExecuteTask(
   if (!status.ok()) {
     if (status.IsIntentionalSystemExit()) {
       return status;
+    } else {
+      RAY_EVENT(ERROR, EL_RAY_CPP_TASK_FAILED)
+              .WithField("task_type", TaskType_Name(task_type))
+              .WithField("function_name", func_name)
+          << "C++ task failed: " << status.ToString();
```

Comment:
```suggestion
          << "Failed to execute the C++ task " << ray_function.GetFunctionDescriptor()->ToString() << ": " << status.ToString();
```

Reply:
We already set func_name the custom field, so here we don't need to print duplicated information to message. And as the unique field,  func_name could represent `FunctionDescriptor`.